api/definitions.yaml

```diff
@@ -233,13 +233,19 @@ ExtendedProfile:
       $ref: "#/PreferredLanguages"
     is_inbox_enabled:
       $ref: "#/IsInboxEnabled"
+    is_webhook_enabled:
+      $ref: "#/IsWebhookEnabled"
     version:
       type: integer
 IsInboxEnabled:
   type: boolean
   default: false
   description: True if the recipient of a message wants to store its content for
     later retrieval.
+IsWebhookEnabled:
+  type: boolean
+  default: false
+  description: True if the recipient of a message wants to receive push notifications.
```

Comment:
`receive push notifications` -> `forward the notifications to the default webhook`